### ISSUE TITLE
LOD Issues

### ISSUE THREAD

**Issue by Sorrien** [LibreVR/RoboRevive#11]
I'm not sure if it's related to the Vive, and how screenspace is defined, but on the Vive the LODs are super aggressive. I'm at LOD 3 at most, with my face 3 inches from a robot's face.

Weapons in my hands are also almost at the lowest LOD.

**Comment by Armada651** [comment]
I can't reproduce this issue on a GTX 1080.

**Comment by Sorrien** [comment]
I should have given more details. I have a GTX 1070, and this is happening in the devkit version (I do not own the release version).

**Comment by Armada651** [comment]
The editor version is only intended for editing, it only has the first mission so it is not a complete game.

Even so I can't reproduce the issue, even in the editor.

**Comment by Sorrien** [comment]
I tried something just to see if I was right about it being the LODs that were messing up. I deleted every LOD except for the most detailed one on the standard issue biped robot, and now the detail for just that robot is forced into LOD 0 (going by appearance). However, when I rip any part of his body off, the piece that comes off must be it's own skeletal mesh, because it goes to one of the lowest LODs immediately. 

Any suggestions?
